src/components/call/Link.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { TranslationFunction } from 'i18next';
+// import { translate } from 'react-i18next';
+import { Issue } from '../../common/model';
+
+interface Props {
+  readonly issue: Issue;
+  readonly t: TranslationFunction;
```

Comment:
This component does not do any translation, so this `prop` and the `TranslationFunction` import is unnecessary.